From f2ed19178fa60e2318b896a85fa2c9ee0c1ea4db Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Tue, 4 Dec 2018 17:13:01 +0000
Subject: [PATCH 343/462] Mantis 0052250: GPPDev: print PDF

- removed compilation in window_IM63
- added environment variables DIR_PDFIX, INC_PDFIX



git-svn-id: svn://localhost/SelfServiceBuild/trunk@607 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/cpp/makefile.mak   | 1 -
 SelfServiceBuild/Build/scripts/setenv.bat | 3 +++
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/SelfServiceBuild/Build/cpp/makefile.mak b/SelfServiceBuild/Build/cpp/makefile.mak
index 9c42fea..dff367d 100644
--- a/SelfServiceBuild/Build/cpp/makefile.mak
+++ b/SelfServiceBuild/Build/cpp/makefile.mak
@@ -110,7 +110,6 @@ PLATFORM_DIRECTORIES= \
  Massai\cpp\ConveyorTest \
  Massai\cpp\GPPDevCustomLib \
  Massai\cpp\GPPDev \
- Massai\cpp\GPPDev\windows_IM63 \
  Massai\cpp\Sel \
  Massai\cpp\ATBTest \
  Massai\cpp\crd\projects\HS \
diff --git a/SelfServiceBuild/Build/scripts/setenv.bat b/SelfServiceBuild/Build/scripts/setenv.bat
index a0e16e6..0f85fde 100644
--- a/SelfServiceBuild/Build/scripts/setenv.bat
+++ b/SelfServiceBuild/Build/scripts/setenv.bat
@@ -500,6 +500,9 @@ set DIR_INFOZIP=%DIR_GLOBALSLIB%\InfoZip\2.3.2\win32
 set INC_INFOZIP=-I%DIR_INFOZIP%\include
 set LIB_INFOZIP=%DIR_INFOZIP%\lib\zip32.lib
 
+set DIR_PDFIX=%DIR_GLOBALSLIB%\pdfix\pdfix_windows_4.1.2
+set INC_PDFIX=-I%DIR_PDFIX%\include\c++
+
 
 rem ######################################################################
 rem # Externe Bibliotheken
-- 
2.41.0.windows.1

